packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts

```diff
@@ -234,10 +244,12 @@ export class FilamentViewer extends LitElement {
       asset.loadResources(resolve, () => {}, basepath(modelUrl), 1);
     });
 
-    loader.delete();
-
     this[$boundingBox] = asset.getBoundingBox();
-    this[$scene].addEntities(asset.getEntities());
+    const entities = asset.getEntities();
+    this[$scene].addEntities(entities);
+    for (const entity of entities) {
+      entity.delete();
```

Comment:
Why do we delete the entities right after adding them?

Reply:
This is an unfortunate emscripten-ism.  The emscripten entity wrapper is being deleted, not the entity itself.

Reply:
ah, good to know. Is it still necessary to also delete them above then, or will they already be gone?

Reply:
As in they are also deleted around line 160.

Reply:
Added a commit with a comment.